server/src/main/java/org/opensearch/action/admin/cluster/stats/ClusterStatsNodes.java

```diff
@@ -854,4 +859,46 @@ public XContentBuilder toXContent(final XContentBuilder builder, final Params pa
 
     }
 
+    static class PeerRecoveryStats implements ToXContentFragment {
+
+        private long total_started_recoveries;
+        private long total_failed_recoveries;
+        private long total_completed_recoveries;
+        private long total_retried_recoveries;
+        private long total_cancelled_recoveries;
+
+        private static final String TotalStartedRecoveries = "total_started_recoveries";
+        private static final String TotalFailedRecoveries = "total_failed_recoveries";
+        private static final String TotalCompletedRecoveries = "total_completed_recoveries";
+        private static final String TotalRetriedRecoveries = "total_retried_recoveries";
+        private static final String TotalCancelledRecoveries = "total_cancelled_recoveries";
```

Comment:
Is there an equation that these fields would always satisfy? For eg - total started would be sum of all remaining fields.
Can we please add details about such constraints and also add appropriate assertions for the same?